manual_jb/content/analyses/indp.md

```diff
@@ -10,26 +10,30 @@ systems subject to budget and operational constraints.
 
 - Implementation: Hesam Talebiyan, Chen Wang, and NCSA IN-CORE Dev Team
 
+**Related publications**
+- Talebiyan, Hesam. "Interdependent Restoration of Infrastructure Networks with Humans in the Loop: decentralized and strategic decision processes." (2021) Diss., Rice University. https://hdl.handle.net/1911/111232.
```

Comment:
To make this hyperlink active in the docs, we need to use the syntax: [https://hdl.handle.net/1911/111232](https://hdl.handle.net/1911/111232)